front/app/components/PageForm/messages.ts

```diff
@@ -42,28 +42,30 @@ export default defineMessages({
     defaultMessage:
       'If you change the URL, links to this page using the old URL will no longer work.',
   },
-  slugRegexError: {
-    id: 'app.components.PagesForm.slugRegexError',
+  slugRegexErrorMessage: {
```

Comment:
Updated error messages in accordance with out new guidelines: https://www.notion.so/citizenlab/Forms-and-validation-b9cdfa1c45274ce2ae7d68405f80cf7c